language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -348,15 +348,15 @@ test("interfaces", async () => {
   }
   const ifaceContract = await aliceLedger.create(buildAndLint.Main.Asset, assetPayload);
   expect(ifaceContract.payload).toEqual(assetPayload);
-  const [ifaceContract1, events1] = await aliceLedger.exercise(buildAndLint.Main.Asset.Transfer, ifaceContract.contractId, {newOwner: BOB_PARTY});
+  const [ifaceContract1, events1] = await aliceLedger.exercise(buildAndLint.Main.Token.Transfer, ifaceContract.contractId, {newOwner: BOB_PARTY});
```

Comment:
can we change the test to send the request and check that we get a failure to make sure the client side can send a request now and so that it blows up once the error from the JSON API changes?

Reply:
done.